emm-docs/content/workflows/working-with-git.md

```diff
@@ -0,0 +1,104 @@
+---
+weight: 2
+---
+Once you have your [project setup]({{< relref "project-setup.md" >}}) with any enabled modules 
+you want to work with, next you will want to create a branch to work on.
+
+## Creating a branch to work on
+
+You may want to run pull to get the latest changes for your base repository in additional to pulling
```

Comment:
[nit] for your base repository. In additional to...

Reply:
done.